app/controllers/v1/live_chat/navigators/invitations_controller.rb

```diff
@@ -70,4 +72,12 @@ def invitation_id
   def not_accepted_invitations
     intervention_load.live_chat_navigator_invitations.not_accepted
   end
+
+  def broadcast_massage(intervention)
+    return if intervention.navigators_count != 1
```

Comment:
Shouldn't it be `intervention.navigators_count.positive?`? What if there are two navigators?

Reply:
If we have two navigators we don't want to send this information. We just only inform FE when we will have a change from 0 to 1. Otherwise, FE has this information after fetching information about the intervention